tests/chainer_tests/functions_tests/test_clipped_relu.py

```diff
@@ -13,7 +13,9 @@
 class TestClippedReLU(unittest.TestCase):
 
     def setUp(self):
-        self.x = numpy.random.uniform(-1, 1, (3, 2)).astype(numpy.float32)
+        x = numpy.random.uniform(-1, 1, (3, 2)).astype(numpy.float32)
+        # Avoid values around zero for stability of numerical gradient
+        self.x = x + (x > 0) - (x <= 0)
         self.gy = numpy.random.uniform(-1, 1, (3, 2)).astype(numpy.float32)
         self.z = 0.75
```

Comment:
Let `z` be greater 1, because elements of x are always greater than 1 when they are positive. It is good to exist both greater and less values then `z` for tests.